composer.json

```diff
@@ -12,7 +12,11 @@
     "maknz/slack": "^1.7",
     "sabre/xml": "^1.4",
     "jarednova/mesh": "0.4",
-    "shortlist-digital/agreable-base-theme": "~2.1"
+    "shortlist-digital/agreable-base-theme": "~2.1",
+    "illuminate/queue": "5.2.*",
+    "illuminate/encryption": "5.2.*",
+    "aws/aws-sdk-php": "dev-master",
```

Comment:
(We tend to dislike `dev-master` as its always bloody updating, and doesn't lock down breaking changes)